src/EnergyPlus/DirectAirManager.hh

```diff
@@ -80,7 +80,7 @@ namespace DirectAirManager {
 	//MODULE VARIABLE DECLARATIONS:
 	extern int NumDirectAir;
 	extern Array1D_bool CheckEquipName;
-
+	extern bool GetDirectAirInputFlag;
```

Comment:
This line can be deleted (again per @mbadams5 comments on https://github.com/NREL/EnergyPlus/commit/d0c373af1b8ce68bd8fd91bc30b4909f09d1c519